### DISCUSSION TITLE
Looking for correct version of nx to migrate to

### DISCUSSION THREAD

**Discussion by nishantg95** [nrwl/nx#6624] [General]
I am updating my Angular app from v 10 to 11, and was wondering what would be the right nx version to migrate to. Is 11.6.3   is the last release that supports Angular 11?

**Comment by brandonroberts** [comment]
Version 12.2.0 was the last version that supports Angular 11. We updated to Angular 12 in the 12.3 release.

https://blog.nrwl.io/incremental-build-improvements-angular-12-distributed-task-execution-and-more-in-nx-12-3-48b5e4722056
